### PR TITLE
Server route client fetch

### PR DESCRIPTION
**Pull request by Nguyephi**

### CODE REVIEW

File: client/src/components/App.js [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
 class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      streamerId: 3
+    }
+  }
+
+  componentDidMount() {
+    const streamerId = this.props.streamerId || this.state.streamerId
```
Comment (kraig-droid) [inline client/src/components/App.js:12]: Nice. I'm going to go do something like this in my code

File: client/src/components/App.js [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+    fetch('/api/livestream/' + streamerId)
```
Comment (kraig-droid) [inline client/src/components/App.js:13]: airbnb eslint will want this as a template string `/api/livestream/${streamerId}`

**Review by kraig-droid — APPROVED** [review]

File: client/src/components/App.js (hunk 2 above)
Comment (Nguyephi) [reply client/src/components/App.js:13]: I noticed that eslint was not registering for some react files. I got to look into that. Thanks for reminding me.

**Review by priscilagonzalez — COMMENTED** [review]
Lgtm.

File: client/src/components/App.js (hunk 2 above)
Comment (Nguyephi) [reply client/src/components/App.js:13]: I found the bug. I need to put the no-plusplus in a string.

**Review by azdoolittle — COMMENTED** [review]
lgtm

**Pull request merged**